lib/generate-report.js

```diff
@@ -183,7 +183,7 @@ function generateReport(options) {
 			feature.isPending = false;
 			suite.featureCount.total++;
 			var idPrefix = staticFilePath ? '' : `${ uuid() }.` ;
-			feature.id = `${ idPrefix }${ feature.id }`.replace(/[^a-zA-Z0-9-_]/g, '-');
+			feature.id = `${ idPrefix }${ feature.id }`.split(" ").join("-");
```

Comment:
Thanks for the PR @Seunghyum 

Sorry for the late reply, i was on vacation. You're fix breaks that fact that strange chars are now allowed again, the previous one didn't allow chars that were not alphanumeric. 

Maybe do something like this
```js
feature.id = `${ idPrefix }${ feature.id }`.split(" ").join("-").replace(/[^a-zA-Z0-9-_]/g, '-');
```